src/share/vm/jfr/jni/jfrJniMethod.cpp

```diff
@@ -303,6 +310,10 @@ JVM_ENTRY_NO_ENV(jboolean, jfr_event_writer_flush(JNIEnv* env, jclass cls, jobje
   return JfrJavaEventWriter::flush(writer, used_size, requested_size, thread);
 JVM_END
 
+JVM_ENTRY_NO_ENV(void, jfr_flush(JNIEnv* env, jobject jvm))
```

Comment:
I'm wondering why jfr entry always has a jvm argument and not use it.

Reply:
it isn't a static method and the argument 'jvm' is the receiver